### PR TITLE
Add 2022q2 OpenSSH update report

### PR DESCRIPTION
**Pull request by emaste**
Minimal OpenSSH report

### CODE REVIEW

File: 2022q2/openssh.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+=== Base System OpenSSH Update
+
+Links: +
+link:https://www.openssh.com/[OpenSSH] URL: link:https://www.openssh.com/[https://www.openssh.com/] +
+link:https://www.openssh.com/txt/release-8.9[OpenSSH 8.9 release notes] URL:https://www.openssh.com/txt/release-8.9[https://www.openssh.com/txt/release-8.9]
```
[the remark this answers is not in the document]
Comment (pauamma) [reply 2022q2/openssh.adoc:5]: Probably, yes.

File: 2022q2/openssh.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+link:https://www.openssh.com/txt/release-9.0[OpenSSH 9.0 release notes] URL:https://www.openssh.com/txt/release-9.0[https://www.openssh.com/txt/release-9.0]
+
+Contact: Ed Maste <emaste@freebsd.org>
+
+OpenSSH, a suite of remote login and file transfer tools, was updated from
+version 8.8p1 to 9.0p1 in the FreeBSD base system.
```
Comment (pauamma) [inline 2022q2/openssh.adoc:11]: May want to mention whether this was/will be MFC'd to 12-stable and 13-stable.

**Review by pauamma — CHANGES_REQUESTED** [review]

**Review by pauamma — APPROVED** [review]
Thanks. Next time, could you avoid force-pushing, as requested in https://github.com/freebsd/freebsd-quarterly#drafts item 5?

**Comment by emaste** [comment]
Will try to remember for next time

**Pull request merged**